src/pages/office/office.ts

```diff
@@ -308,16 +307,19 @@ export class OfficePage extends BasePageComponent {
     this.store$.dispatch(new AdditionalInformationChanged(additionalInformation));
   }
 
-  dangerousFaultCommentChanged(dangerousFaultComment: FaultComment) {
-    this.store$.dispatch(new AddDangerousFaultComment(dangerousFaultComment.competency, dangerousFaultComment.comment));
+  dangerousFaultCommentChanged(dangerousFaultComment: CommentedCompetency) {
+    const a = new AddDangerousFaultComment(dangerousFaultComment.competencyIdentifier, dangerousFaultComment.comment);
```

Comment:
I don't really see too much value in extracting the action into a variable which has a non descriptive name.

Reply:
I agree totally, this was purely to keep the linter happy due, the line was too long.

Reply:
Break the line, or disable the linting rule. But changing the structure of the code because of linting is not very good in my opinion.

Reply:
Fixed.